src/main/java/com/transportcompany/transportcompany/models/entities/Vehicle.java

```diff
@@ -0,0 +1,16 @@
+package com.transportcompany.transportcompany.models.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table (name = "vehicle")
+public class Vehicle {
```

Comment:
You had Drivers and Orders entity in plural, but this is different. Why? We should have consistency.
Please keep in mind that the table should use plural and the entity should use singular.